scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/package.scala

```diff
@@ -10,6 +10,8 @@ package object tokenizers {
     "case",
     "do",
     "else",
+    "enum",
+    "export",
```

Comment:
what about `using`, `extension`, `derives` and all the other scala3 innovations?

Reply:
Also there are these: https://dotty.epfl.ch/docs/reference/soft-modifier.html
But I think I will test these separately and provide examples in a separate issue if needed

Reply:
@vigoo @tgodzik so all hard keywords are covered, then? if so, lgtm.

Reply:
Based on this:
https://dotty.epfl.ch/docs/internals/syntax.html#regular-keywords

i think these two are missing:
```
then
?=>
```

Reply:
Those are soft keywords and I think we covered most of the problematic cases in tests. Inline and extension being the the hardest. Let me check if it will not break the Scala 2 code actually.

Reply:
> i think these two are missing:
> 
> ```
> ?=>
> ```

wouldn't this be covered as a non-alphanum identifier?

Reply:
> wouldn't this be covered as a non-alphanum identifier?

Well 

```scala
val ?=> = 0 // invalid
val `?=>` = 0 // invalid
val ?==> = 0 // valid
```

i think it behaves exactly as the other keywords in this regard

Reply:
interesting :) let's add it, then.

separately, I'd sort these keywords, hard to find whether something is already there.

Reply:
Added both and also changed it to make sure we only backtick those keywords with a proper dialect option on.